java/configuration/src/main/java/sleeper/configuration/properties/instance/GarbageCollectionProperty.java

```diff
@@ -27,11 +27,18 @@ public interface GarbageCollectionProperty {
             .defaultValue("15")
             .propertyGroup(InstancePropertyGroup.GARBAGE_COLLECTOR)
             .runCdkDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty GARBAGE_COLLECTOR_LAMBDA_TIMEOUT_IN_MINUTES = Index.propertyBuilder("sleeper.gc.lambda.timeout.minutes")
+            .description("The configurable timeout wait (minutes) for the garbage collector lambda. When not set defaults to the garbage collector period.")
+            .defaultValue("15")
```

Comment:
It might be best to default this to lower than the GC period, maybe 14 minutes?

I suppose it depends on the conditions under which a message would end up on the dead letter queue. It might be good if we got a message on the dead letter queue every time the lambda times out. I'm not sure if that will happen if the lambda times out?